dashboard/app/models/pd/workshop_material_order.rb

```diff
@@ -47,7 +53,10 @@ class WorkshopMaterialOrder < ActiveRecord::Base
     validates_presence_of :zip_code
     validates_presence_of :phone_number
     validates_inclusion_of :state, in: STATE_ABBR_WITH_DC_HASH.keys.map(&:to_s), if: -> {state.present?}
-    validates :phone_number, us_phone_number: true, if: -> {phone_number.present?}
+
+    # Make sure the phone number contains at least 10 digits.
+    # Allow any format and additional text, such as extensions.
+    validates :phone_number, format: /(\d.?){10}/, if: -> {phone_number.present?}
```

Comment:
Negative. The () is a grouping, not an exact match. This matches the group exactly 10 times, where the group consists of a single digit `\d`, followed optionally by another character `.?`. Though I do now realize this should be `.*` instead of ?, so it can match 0 or more other characters instead of exactly 0 or 1. I'll update that.

Anyway, see:
```
irb(main):001:0> r = /(\d.*){10}/
=> /(\d.?){10}/
irb(main):002:0> '((((((((((' =~ r
=> nil
irb(main):003:0> '1234567890' =~ r
=> 0
irb(main):012:0> '123  4567890' =~ r  # this would have failed (returned nil) with the .? regex
=> 0
```

FYI You can easily test ruby regexes at http://rubular.com/